app/javascript/articles/components/index.jsx

```diff
@@ -0,0 +1,7 @@
+export * from './ContentTitle';
```

Comment:
All these mini components were in Article.jsx originally, but codeclimate complained about file size. I've rerun the tests and it still matches the snapshots and Storybook stories are looking good.